apps/judicial-system/backend/src/app/modules/notification/services/caseNotification/caseNotification.service.ts

```diff
@@ -1299,14 +1355,14 @@ export class CaseNotificationService extends BaseNotificationService {
       theCase.courtCaseNumber,
     )
 
-    return this.sendEmail(
+    return this.sendEmail({
       subject,
       html,
-      this.formatMessage(notifications.emailNames.prison),
-      this.config.email.prisonEmail,
-      undefined,
-      true,
-    )
+      recipientName: this.formatMessage(notifications.emailNames.prison),
+      recipientEmail: this.config.email.prisonEmail,
+      attachments: undefined,
```

Comment:
Consider removing undefined properties as they are no longer needed when passing in an object as a parameter.